README.md

```diff
@@ -137,6 +140,9 @@ heading = "h3"
 link = "h3 a"
 # Optional CSS selector relative to `item` that will supply the content of the RSS item.
 summary = ".post-body"
+# Optional CSS selector relative to `item` that supplies a media (image, video..) content to be added as an RSS enclosure.
+# The media link must be given by the `src` attribute of the selected element.
+media = ".example-img > a"
```

Comment:
This example should be tweaked (probably to something like `media = ".example-img > img"` as `a` tags don't have a `src` attribute for the URL.

I think this exposes a limitation of the current implementation. It seems reasonable for this feature to be used to match on at least `video`, `audio`, `img`, and `a` elements. As it is it will work for the first three, but not `a`. Doesn't need to be a part of this PR but a natural extension would be to fall back on the `href` attribute if `src` is missing.